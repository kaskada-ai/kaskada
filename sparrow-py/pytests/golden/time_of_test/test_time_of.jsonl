{"_time":851013597000000000,"_subsort":0,"_key_hash":12960666915911099378,"_key":"A","m":5.0,"time_of_m":851013597000000000,"n":10.0,"time_of_n":851013597000000000}
{"_time":851013598000000000,"_subsort":1,"_key_hash":2867199309159137213,"_key":"B","m":24.0,"time_of_m":851013598000000000,"n":3.0,"time_of_n":851013598000000000}
{"_time":851013599000000000,"_subsort":2,"_key_hash":12960666915911099378,"_key":"A","m":17.0,"time_of_m":851013599000000000,"n":6.0,"time_of_n":851013599000000000}
{"_time":851013600000000000,"_subsort":3,"_key_hash":12960666915911099378,"_key":"A","m":null,"time_of_m":851013600000000000,"n":9.0,"time_of_n":851013600000000000}
{"_time":851013601000000000,"_subsort":4,"_key_hash":12960666915911099378,"_key":"A","m":12.0,"time_of_m":851013601000000000,"n":null,"time_of_n":851013601000000000}
{"_time":851013602000000000,"_subsort":5,"_key_hash":12960666915911099378,"_key":"A","m":null,"time_of_m":851013602000000000,"n":null,"time_of_n":851013602000000000}
