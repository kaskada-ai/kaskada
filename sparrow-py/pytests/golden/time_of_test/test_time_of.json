{"_time":851042397000000000,"_subsort":0,"_key_hash":12960666915911099378,"_key":"A","m":5.0,"time_of_m":851042397000000000,"n":10.0,"time_of_n":851042397000000000}
{"_time":851042398000000000,"_subsort":1,"_key_hash":2867199309159137213,"_key":"B","m":24.0,"time_of_m":851042398000000000,"n":3.0,"time_of_n":851042398000000000}
{"_time":851042399000000000,"_subsort":2,"_key_hash":12960666915911099378,"_key":"A","m":17.0,"time_of_m":851042399000000000,"n":6.0,"time_of_n":851042399000000000}
{"_time":851042400000000000,"_subsort":3,"_key_hash":12960666915911099378,"_key":"A","m":null,"time_of_m":851042400000000000,"n":9.0,"time_of_n":851042400000000000}
{"_time":851042401000000000,"_subsort":4,"_key_hash":12960666915911099378,"_key":"A","m":12.0,"time_of_m":851042401000000000,"n":null,"time_of_n":851042401000000000}
{"_time":851042402000000000,"_subsort":5,"_key_hash":12960666915911099378,"_key":"A","m":null,"time_of_m":851042402000000000,"n":null,"time_of_n":851042402000000000}
